apps/src/javalab/Javalab.js

```diff
@@ -145,7 +145,8 @@ Javalab.prototype.init = function (config) {
         this.onOutputMessage,
         this.onNewlineMessage,
         this.setIsRunning,
-        STATUS_MESSAGE_PREFIX
+        STATUS_MESSAGE_PREFIX,
+        this.onOutputMessage
```

Comment:
Could add a comment that `javalab` does not assume message ends in newline so that the `onPartialOutputMessage` param is set to same function as `onOutputMessage` is.

Reply:
done!